### PR TITLE
Replace `repeatOnAjax` with explicit `onetime`

### PR DESCRIPTION
**Pull request by fregante**
Waiting for https://github.com/sindresorhus/onetime/issues/12

Can be reviewed. Doesn't need testing but it's worth a look just in case features _need_ that old specific behavior (unlikely)

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -68,11 +68,8 @@ features.add({
 	/** Whether to wait for DOM ready before runnin `init`. `false` makes `init` run right as soon as `body` is found. @default true */
 	waitForDomReady: false,
 
-	/** Whether to re-run `init` on pages loaded via AJAX. @default true */
-	repeatOnAjax: false,
```
Comment (yakov116) [inline contributing.md:72]: Want to give an example of using oneTime here?

Comment (fregante) [reply contributing.md:72]: Thought about it, but it's not necessary

**Comment by sindresorhus** [comment]
What's the benefit of this change?

**Comment by fregante** [comment]
Sorry, I didn't add context:
https://github.com/sindresorhus/refined-github/pull/3314#discussion_r453770549
https://github.com/sindresorhus/refined-github/pull/3412#pullrequestreview-457547068

In short, pages can change via ajax, but if `repeatOnAjax === false` RG won't call their `init` even if now their `include` conditions would match. 

That logic was never correct; our real intent was only to prevent duplicate execution on the same document, hence the `onetime`

**Comment by sindresorhus** [comment]
Makes sense. I would probably have added a `initOnce` option or something, since it's used so much.

**Pull request merged**